### DISCUSSION TITLE
Compatibilität für ältere Android Tablets

### DISCUSSION THREAD

**Discussion by softwarecrash** [Zefau/ioBroker.jarvis#758] [Q&A]
gibt es eine möglichkeit irgendwie eine version oder modus oder dergleichen einzubauen damit ältere android tablets / Handys mit jarvis funktionieren?
die geräte sind billig zu haben, aber nicht für jedes gibt es ein Custom rom um an eine aktualisierte webview zu kommen.
Geräte mit android 4.2 / 4.4 sind nicht dazu zu bewegen jarvis überhaupt anzuzeigen.
Danke

**Comment by Zefau** [comment]
Was für einen Browser in welcher Version nutzt du auf den Geräten? Was genau passiert, wenn du jarvis startest?

Die Limitierung ist nicht jarvis selbst, sondern socket.io, vermute ich. Das funktioniert auf älteren Geräte nicht.

**Reply by Zefau** [reply]
https://developer.mozilla.org/en-US/docs/Web/API/WebSocket#browser_compatibility
